spec/compiler/macro/macro_methods_spec.cr

```diff
@@ -3692,10 +3692,9 @@ module Crystal
       end
 
       it "reads file (doesn't exist)" do
-        assert_error <<-CRYSTAL,
+        assert_error <<-CRYSTAL
           {{read_file("#{__DIR__}/../data/build_foo")}}
           CRYSTAL
-          "No such file or directory"
```

Comment:
**suggestion:** We could test for the common part of the error message, maybe `Error opening file with mode 'r'`?